KNOWN-ISSUES.md

```diff
@@ -0,0 +1,19 @@
+# v1.0-rc3
+
+The issues are mainly in the following areas
+
+* No proper background support on both Android and iOS.
+* Edges cases are not handled properly, such as mic/camera permission and no network.
+* Sometimes camera and mic do not work after re-joining the meeting.
+
+Not all the issues are listed there. They include
+
+* Individual device or model-specified issues
```

Comment:
change model-specified to  model-specific

Reply:
Done